app/models/conference.rb

```diff
@@ -858,7 +858,7 @@ def cfp_set?
   # * +True+ -> If conference has a start and a end date.
   # * +False+ -> If conference has no start or end date.
   def registration_date_set?
-    !!registration_period && !!registration_period.start_date && !!registration_period.end_date
+    registration_period && registration_period.start_date && registration_period.end_date
```

Comment:
@Ana06 I ran rspec locally with 
`registration_period && registration_period.start_date.present? && registration_period.end_date.present?`. Test `Conference#get_status calculates correct for new conference` failed with above configuration.